web/app.js

```diff
@@ -372,6 +372,9 @@ const PDFViewerApplication = {
           params.get("supportscaretbrowsingmode") === "true"
         );
       }
+      if (params.has("spreadmode")) {
+        AppOptions.set("spreadModeOnLoad", parseInt(params.get("spreadmode")));
```

Comment:
Can we please use consistent names here?
```suggestion
      if (params.has("spreadmodeonload")) {
        AppOptions.set("spreadModeOnLoad", parseInt(params.get("spreadmodeonload")));
```